packages/core/content-manager/admin/src/hooks/useDocument.ts

```diff
@@ -196,5 +205,61 @@ const useDoc = () => {
   };
 };
 
-export { useDocument, useDoc };
+/**
+ * @public
+ * @experimental
+ * Content manager context hooks for plugin development.
+ * Make sure to use this hook inside the content manager.
+ */
+const useContentManagerContext = () => {
```

Comment:
Maybe we should remove the useDoc hook now?

- It feels weird to have 3 levels of abstraction (useContentManagerContext calls useDoc calls useDocument)
- The useDoc / useDocument naming is confusing
- Everything it returns is returned from this hook

So I think we could move its contents into this hook

Reply:
Honestly I don't want to change any of the internal or refactor anything right now it's really not the goal. 
I Just want to expose a convenient hook for plugin developers. We can change all our internals afterwards for sure. I agree with your points :)

Reply:
yes it doesn't need to be now, we can simplify it later